crates/red_knot_python_semantic/src/types/infer.rs

```diff
@@ -124,7 +124,6 @@ pub(crate) fn infer_definition_types<'db>(
     let file = definition.file(db);
     let _span = tracing::trace_span!(
         "infer_definition_types",
-        definition = ?definition.as_id(),
```

Comment:
I don't think the definition id gives us much here and it resulted in very long tracing lines but I'll revert if this ends up being controverisal